docs/quickstart.md

```diff
@@ -0,0 +1,347 @@
+## Usage and Deployment
+
+### Prerequisites
+
+> [!IMPORTANT]
+> Before installing the LocalPV-ZFS driver please make sure your Kubernetes Cluster meets the following prerequisites:
+> 1. All the nodes must have ZFS utils package installed
+> 2. A ZPOOL has been configurred for provisioning volumes
+> 3. You have access to install RBAC components into kube-system namespace. The OpenEBS ZFS driver components are installed in kube-system namespace to allow them to be flagged as system critical components.
```

Comment:
Why are these system-critical components? Is this step mandatory?

considered instead of flagged

Reply:
Okay that's no longer the case, we no longer do it in kube-system. I will remove this statement altogether.